### PR TITLE
Language and minor lore update

### PR DESCRIPTION
**Pull request by MefAnneFeminene**
## About The Pull Request
This PR changes the name of the Sol Flag in the loadout (I forgor to do it) and, more largely, changes the name and description of Pan-Slavic to Interslavic and a more lore-friendly description. Sol Common was also changed to Galactic Common with a fitting description as well as new syllables to fit the new base language.

## Why It's Good For The Game
Lore update from the director himself! <3

## Changelog

:cl:
add: Galactic Common language
add: Interslavic Language
del: Pan-Slavic Language
del: Sol Common language
fix: Loadout name for Sol Flag
/:cl:

### CODE REVIEW

File: modular_zubbers/code/modules/loadouts/loadout_items/loadout_datum_inhands.dm [hunk 1]
```diff
@@ -15,3 +15,7 @@
 /datum/loadout_item/inhand/flag_moghes //sprites by Crumpaloo
 	name = "Folded Tizirian Empire Flag"
 	item_path = /obj/item/sign/flag/tizira
+
+/datum/loadout_item/inhand/flag_solfed
+	name = "Folded Solarian State Flag"
+	item_path = /obj/item/sign/flag/terragov
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/loadouts/loadout_items/loadout_datum_inhands.dm:21]:
```suggestion
```

File: modular_zubbers/modules/customization/modules/language/common.dm [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+	space_chance = 60
+	syllables = list(
+		"at", "bo", "cre", "del", "ei", "fir", "guh", "ho", "ill", "jo", "kat", "le", "mon", "ne", "op", "po", "que", "ro", "set", "tin", "um", "ven", "wir", "xa", "yo", "zet",
+		"ao", "bar", "co", "don", "ef", "fo", "gon", "hi", "io", "jit", "kir", "lol", "mit", "no", "oh", "pa", "quo", "rex", "so", "ta", "ur", "va", "wa", "xi", "ye", "zat",
+		"ah", "be", "ch", "da", "eu", "fu", "ga", "ha", "in", "ja", "ko", "li", "ma", "nya", "oi", "per", "qe", "ri", "sex", "tic", "uo", "vo", "wu", "xe", "yr", "ze",
+		"'al", "'bel", "'cal", "'do", "'el", "'feb", "'gol", "'hir", "'ib", "'jah", "'kil", "'lo", "'mo", "'no", "'ol", "'pek", "'qre", "'rib", "'sif", "'tal", "'ui", "'vik", "'wo", "'xin", "'yef", "'zak",
+		"ap", "br", "cze", "di", "eck", "fu", "gup", "he", "ir", "je", "ka", "la", "mer", "na", "os", "pi", "qux", "rum", "sin", "tum", "ut", "vye", "weh", "xje", "yoh", "zog"
+	)
+	icon_state = "solcommon"
+	icon = 'modular_skyrat/master_files/icons/misc/language.dmi'
```
Comment (Majkl-J) [inline modular_zubbers/modules/customization/modules/language/common.dm:13]:
```suggestion
```

**Review by Majkl-J — CHANGES_REQUESTED** [review]
You don't have to copy everything, just the stuff you are overriding. Otherwise good

File: modular_zubbers/modules/customization/modules/language/common.dm (hunk 2 above)
Comment (MefAnneFeminene) [reply modular_zubbers/modules/customization/modules/language/common.dm]: This removes changes actually changed to the file. I removed the repetitive changes and re-added the syllables list (which was changed)

**Review by Majkl-J — APPROVED** [review]
All good

**Pull request merged**